scalahost/sbt/src/sbt-test/sbt-scalahost/semantic-example/app/src/test/scala/MainTest.scala

```diff
@@ -0,0 +1,74 @@
+import scala.meta._
+import scala.meta.internal.semantic.{vfs => v}
+import scala.meta.internal.semantic.{schema => s}
+import java.nio.file._
+import org.scalatest._
+
+abstract class BaseMirrorSuite extends FunSuite {
+  implicit val mirror = Mirror()
+  org.scalameta.logger.elem(mirror.database, mirror.sources)
+  def checkContents(f: String => Unit): Unit = checkDb { case (_, x) => f(x.contents) }
+  def checkDb(f: (RelativePath, s.Attributes) => Unit): Unit = {
+    val db = v.Database.load(Classpath(sys.props("scalameta.classpath"))).toSchema
+    assert(db.entries.nonEmpty, s"$db.entries.nonEmpty")
+    db.entries.foreach { case (path, attributes) => f(path, attributes) }
+  }
+
+  def assertNonEmptyMirror(): Unit = {
+    test("Mirror.sources") {
+      assert(mirror.sources.length == 3)
+    }
+    val paths = List(
+      "library1/target/scala-2.11/classes/META-INF/semanticdb/library1/src/main/scala-2.11/Test.semanticdb",
+      "library2/target/scala-2.11/classes/META-INF/semanticdb/library2/src/main/scala/Test.semanticdb",
+      "library2/target/scala-2.11/test-classes/META-INF/semanticdb/library2/src/test/scala/TestMain.semanticdb"
+    )
+    // parent because fork := true
+    val workingDir = Paths.get(sys.props("user.dir")).getParent
```

Comment:
I haven't. But these tests run via scripted so the overhead of forking here is negligible compared to scripted slowness.